examples/lib/libcapture/Makefile.example.in

```diff
@@ -0,0 +1,16 @@
+LIBSURICATA_CONFIG ?=	@CONFIGURE_PREFIX@/bin/libsuricata-config
+
+SURICATA_LIBS =		`$(LIBSURICATA_CONFIG) --libs --static`
+SURICATA_CFLAGS :=	`$(LIBSURICATA_CONFIG) --cflags`
+
+# Currently the Suricata logging system requires this to be even for
+# plugins.
+CPPFLAGS +=    "-D__SCFILENAME__=\"$(*F)\""
+
+all: libcapture
```

Comment:
how does this work wrt name spaces? I feel this is a very generic name that may exist in the system or that we might want to use for non-example code as well later

Reply:
Not sure I follow. This is an example program, not meant to be installed anywhere.

Reply:
Guess I would expect something more specific, like libsuricata_example_capture or something. Not something generic.

Reply:
Pretty close already no?  `examples/lib/libcapture`? Seems more like stuff for the readme.